firefox-ios/Client/Frontend/Browser/TabScrollController.swift

```diff
@@ -15,6 +15,8 @@ final class TabScrollController: NSObject,
         static let toolbarBaseAnimationDuration: CGFloat = 0.2
         static let minimalAddressBarAnimationDuration: CGFloat = 0.4
         static let heightOffset: CGFloat = 14
+        static let minimumScrollThreshold: CGFloat = 20
+        static let minimumScrollVelocity: CGFloat = 100
```

Comment:
we might need 2 values for velocity I feel that hiding the toolbar is not that bad but to show it again feels like I need to scroll faster